### ISSUE TITLE
Search by comments

### ISSUE THREAD

**Issue by AlexWayfer** [getsentry/sentry#14013]
## Summary

I want to find issues by text from comments.

## Motivation

I don't remember what URL of issue is, but I remember that I referred to PR for library on GitHub.

## Additional Context

https://github.com/fnando/browser/pull/351

**Comment by BYK** [comment]
Is this a feature request? Bug report? For which offering (SaaS or on-premise)?

**Comment by AlexWayfer** [comment]
> Is this a feature request? Bug report? 

It was created from `Feature request` template.

> For which offering (SaaS or on-premise)?

Does it matter? We're using self-hosted.

**Comment by BYK** [comment]
>It was created from Feature request template.

As you can see from the issue itself, the template selected is not shown. I'll go ahead and improve the feature request template to make it clearer.

>Does it matter? We're using self-hosted.

Yes, it does as SaaS goes ahead of on-premise releases and this may already be available there (which means it will come to on-premise). Will check with the team.

**Comment by AlexWayfer** [comment]
> As you can see from the issue itself, the template selected is not shown. 

Yes.

> I'll go ahead and improve the feature request template to make it clearer.

Thank you.

> Yes, it does as SaaS goes ahead of on-premise releases and this may already be available there (which means it will come to on-premise). Will check with the team.

Oh, OK. Thank you. I think, it can be added to the template too.

**Comment by AlexWayfer** [comment]
> This issue has gone three weeks without activity. In another week, I will close it.
> 
> But! If you comment or otherwise update it, I will reset the clock, and if you label it `Status: Accepted`, I will leave it alone ... forever!
> 
> "A weed is but an unloved flower." ― _Ella Wheeler Wilcox_ 🥀

This functionality still can be very helpful.

**Comment by blackthornedk** [comment]
I second this!
I'm in the process of triaging and creating tickets, and would like to find all Sentry's, which does *not* have a ticket ID in a comment.

**Comment by BYK** [comment]
This requires non-trivial effort to implement as far as I'm aware. I'll let @fpacifici comment more on this (and possibly @wedamija?)

**Comment by wedamija** [comment]
I don't think this is something we can support right now. It's possible that some time in the future we could put comments into Snuba and then they'd be searchable, but not sure when we'd be able to fit this on the roadmap.

**Comment by johnnybaloney** [comment]
Are there any news in this area?
